nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/events/BulletinFactory.java

```diff
@@ -27,30 +27,28 @@
 
     private static final AtomicLong currentId = new AtomicLong(0);
 
+    private BulletinFactory() {
+    }
+
     public static Bulletin createBulletin(final Connectable connectable, final String category, final String severity, final String message) {
         final ComponentType type;
```

Comment:
Is a good idea to break that into its own method - would make sense then to also collapse the declaration and assignment into a single statement as well now. (I.e., `final ComponentType type = getComponentType(connectable);`)